swap/src/cli/command.rs

```diff
@@ -111,15 +108,13 @@ pub enum Command {
 pub struct AliceConnectParams {
     #[structopt(
         long = "seller-peer-id",
```

Comment:
Now that we are actively exposing the user to these options, should we rename them to `maker-peer-id` in hindsight of the next project which is going to add `sell-xmr` as a command?

Reply:
I would change this when sell-xmr is added. The changelog entry to describe this change will be easier to understand when it is combined with the sell xmr feature addition